index.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+const db = require("./data/models/dataModel.js");
+const server = express();
+const port = 9000;
+
+//apply middleware
+server.use(express.json());
+
+server.get('/', (req, res) => {
+	res.send('<h2>Server is running.</h2>');
+})
+
+
+//--------- DISH ROUTES -------
+server.get("/api/dishes", (req, res) => {
+    db.getDishes()
+      .then(dishes => res.status(200).json(dishes))
+      .catch(err => res.send(err));
+  });
+  
+  server.post("/api/dishes", (req, res) => {
+    const dish = req.body;
+    console.log(dish);
+    db.addDish(dish)
+      .then(id => res.status(200).json(id))
+      .catch(err => res.status(200).json(err));
+  });
+  
+  server.get("/api/dishes/:id", (req, res) => {
+    id = req.params.id;
+    if (!id) {
+      res.status(400).json({ error: "Bad request" });
+    }
```

Comment:
This check for `id` will never fail since it is required to reach this endpoint. Check to see if the `id` exists in the database instead.